cmd/containerd-shim-runhcs-v1/service_internal.go

```diff
@@ -82,6 +82,8 @@ func (s *service) createInternal(ctx context.Context, req *task.CreateTaskReques
 		logrus.SetLevel(logrus.DebugLevel)
 	}
 
+	setupDebuggerEvent()
```

Comment:
Lets do this very first in the create call. So we could debug the opts deserialization too.

Reply:
Deal. Updated.